src/lightkurve/io/eleanorlite.py

```diff
@@ -0,0 +1,94 @@
+"""Reader for GSFC-ELEANOR-LITE light curve files. 
+Details can be found at https://archive.stsci.edu/hlsp/eleanor and https://archive.stsci.edu/hlsp/gsfc-eleanor-lite
+"""
+from ..lightcurve import TessLightCurve
+from ..utils import TessQualityFlags
+from astropy import units as u
+
+from .generic import read_generic_lightcurve
+
+def read_eleanorlite_lightcurve(filename,
+    time_column="TIME",
+    flux_column="CORR_FLUX", 
+    flux_err_column="FLUX_ERR", 
+    cadenceno_column="FFIINDEX",
+    centroid_col_column="X_CENTROID",
+    centroid_row_column="Y_CENTROID",
+    quality_column="QUALITY",
+    quality_bitmask=None, 
+    **kwargs):
+    """Returns a `TessLightCurve` object given a light curve file from the GSFC Eleanor-lite Pipeline.
+
+    By default, eleanor's `CORR_FLUX` column is used to populate the `flux` values,
+    and 'FLUX_ERR' is used to populate `flux_err`. Note that the "FLUX_ERR" column in the 
+    Eleanor FITS file is referred to the uncertainty of "RAW_FLUX", not "CORR_FLUX"
```

Comment:
The comment about `flux_err` is no longer accurate after the changes.

Reply:
Good catch, thanks! It's fixed